Cargo.toml

```diff
@@ -27,6 +27,7 @@ cfg-if = "1.0.0"
 libc = "0.2"
 num_cpus = "1.0"
 paste = "1.0.14"
+strum = { version = "0.25.0", features = ["derive"] }
```

Comment:
What? Didn't you already add strum as a dependency in an early PR for generating enum counts?